source/_patterns/01-elements/loading-indicator/loading-indicator.scss

```diff
@@ -34,8 +34,6 @@
 	@include loadingindicator--sizes("elm-");
 	@extend %size-m; // default
 
-	animation: loadingindicator-rotate 3s linear infinite;
-
```

Comment:
@nmerget what was the reason to remove this declaration ?

Reply:
It is a duplicate from line 31